x-pack/plugins/enterprise_search/public/applications/app_search/components/schema/views/schema.tsx

```diff
@@ -38,13 +44,24 @@ export const Schema: React.FC = () => {
           { defaultMessage: 'Add new fields or change the types of existing ones.' }
         )}
         rightSideItems={[
-          <EuiButton fill>
+          <EuiButton
+            fill
+            disabled={!hasSchemaChanged}
+            isLoading={isUpdating}
```

Comment:
As noted in the previous logic PR (https://github.com/elastic/kibana/pull/99548#discussion_r627764850), the `isLoading` state for these action buttons is also new from the standalone UI. It's already me prevented from spam clicking about a million times, and feels nice and responsive.